src/meta/src/manager/env.rs

```diff
@@ -350,6 +350,7 @@ impl MetaSrvEnv {
         init_system_params: SystemParams,
         init_session_config: SessionConfig,
         meta_store_impl: MetaStoreImpl,
+        is_sql_backend_cluster_first_launch: bool,
```

Comment:
Passing this only for sql backend seems a littble bit odd. Can we either move the `Migrator::up` for sql backend inside `MetaSrvEnv::new` or move the `cluster_first_launch` check for etcd metastore outside `MetaSrvEnv::new` to make the logic more consistent for both backends?